docs/achievements.md

```diff
@@ -0,0 +1,64 @@
+# Achievements System
+
+## Initial Setup
+
+Initialise the achievements system with an array of achievement objects.  
+It is recommended to load this data from a JSON file.
+
+`gmi.achievements.init(achievementsData);`
+
+## Configuration
+
+For each achievement, create an object with a key, name and description.
+
+```json
+{
+    "key": "first_win",
+    "name": "Just Getting Started",
+    "description": "You managed to crack your first egg in time.",
+    "points": 100
+}
+```
+**Required Parameters:**
+* **key** [string a-z0-9_] a unique (per game) identifier, lowercase alphanumeric with underscore
+* **name** [string] This is the human readable name that will be displayed in the achievements list.
+* **description** [string] Description that will be displayed in the achievements list.
+* **points** [integer] Currently unused but required for future use. Should add up to 1000 points per game but otherwise can be weighted for difficulty or designer preference.
+
+**Optional Parameters:**
+* **maxProgress** [integer] Enables the progress bar and sets its limit.
+* **position** [string] This is the position that specifies where the achievements notification should show. Can be either "top" or "bottom" - defaults to "bottom".
+* **additional** {prefix [string], text [string]} Add a secondary text element to the description.
+  - The prefix of this will be in bold. e.g:
+`"additional": {"prefix": "hint", "text": "You can find these items on your travels."}`
+will add: "**hint:** You can find these items on your travels" to the description text.
+
+#### String Lengths
+Achievement names and descriptions should be an appropriate length so as not to be cut off, or for the text to overflow out of the notification box that appears in-game when a user earns an achievement.
```

Comment:
Do we have any known or estimated character limits for any of the fields? I wouldve thought UX would have an idea?

Reply:
I asked and they said there's no estimated/known character limit because it depends on what characters you use. If it's too big it will show "..." at the end